azimuth/startup.py

```diff
@@ -120,6 +120,9 @@ class Startup:
         dependency_names=["prediction"],
         run_on_all_pipelines=True,
     ),
+]
+
+LAST_TASKS = [
```

Comment:
As discussed offline, your comment already makes it clear that the order is important, and we can't rename all those constants with names that indicate in which order they should be added, so I don't think it's necessary for this one either. We could go with something more similar to the other ones:
```suggestion
BY_FILTER_TASKS = [
```

Reply:
Sounds good!